src/winrtble/mod.rs

```diff
@@ -17,3 +17,17 @@ mod ble;
 pub mod manager;
 pub mod peripheral;
 pub mod utils;
+
+#[non_exhaustive]
+struct AdvertisementDataType;
+
+// Only some of the assigned numbers are populated here.
+// Feel free to add them as needed from https://www.bluetooth.com/specifications/assigned-numbers/generic-access-profile/
+impl AdvertisementDataType {
+    ///Core Specification Supplement, Part A, section 1.11
+    pub const SERVICE_DATA_16_BIT_UUID: u8 = 0x16;
+    ///Core Specification Supplement, Part A, section 1.11
+    pub const SERVICE_DATA_32_BIT_UUID: u8 = 0x20;
+    ///Core Specification Supplement, Part A, section 1.11
```

Comment:
Nit: Having the same doc comment on all three constants is a bit repetitive, maybe it belongs on the module instead?

Reply:
it was copy/pasted directly from the assigned numbers. I may just leave a module level comment saying to find out more information there